File: springbootjparest/src/main/java/com/scarc/springbootjparest/models/Mail.java
package com.scarc.springbootjparest.models;

import lombok.*;

import javax.persistence.*;
import java.io.File;
import java.time.LocalDate;
import java.time.LocalTime;
import java.util.List;

@Data
@Entity
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Mail {
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  private Long id;

  private int nr;
  private LocalDate date;
  private LocalTime time;
  private String subject;
  private String content;
  private File attachment1;
  private File attachment2;
  private File attachment3;

  @ManyToOne
  private Admin sender;

  @EqualsAndHashCode.Exclude
  @OneToMany(mappedBy = "mail", orphanRemoval = true, cascade = CascadeType.ALL)
  private List<Responsible> receivers;

}

File: KOTLIN/Wirtschaftstag/Kotlin_Wirtschaftstag/androidApp/src/main/java/me/scarc/androidApp/data/Mail.kt
package me.scarc.androidApp.data

import java.io.File
import java.time.LocalDate
import java.time.LocalTime
import java.util.*

data class Mail(
    var id: Int,
    var nr: Int,
    var date: LocalDate,
    var time: LocalTime,
    var subject: String,
    var content: String,
    var attachment1: File,
    var attachment2: File,
    var attachment3: File,
    var recipients: List<Responsible>,
    var sender: Admin
) {
}